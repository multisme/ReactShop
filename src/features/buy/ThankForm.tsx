import { useEffect } from "react";
import { useDispatch } from "react-redux";
import { useHistory } from "react-router-dom";

import { emptyCart } from "features/cart/cartSlice";

const ThankForm = () => {
  const history = useHistory();
  const dispatch = useDispatch();

  useEffect(() => {
    dispatch(emptyCart);
  });
  setTimeout(() => {
    history.push("./");
  }, 5000);
  return (
    <div className="rectangle">
      <h3>Thank You</h3>
    </div>
  );
};

export default ThankForm;
