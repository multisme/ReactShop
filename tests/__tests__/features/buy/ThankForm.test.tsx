import {shallow} from "enzyme"
import ThankForm from "features/buy/ThankForm"

describe("ThankForm", () =>{
        
        it("rendets", () => {
                const component = shallow(<ThankForm />);
                expect(component).toMatchSnapshot()
        })
})
