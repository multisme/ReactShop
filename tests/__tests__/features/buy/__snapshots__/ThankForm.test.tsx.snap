// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ThankForm rendets 1`] = `
<div
  className="rectangle"
>
  <h3>
    Thank You
  </h3>
</div>
`;
